src/__tests__/utils.test.ts

```diff
@@ -15,166 +15,167 @@ import * as path from "path";
-import { checkFilterString, createDummyDataset, createPhysicalDocument, generateTempFileName,
-   validateDatasetName, validateMemberName } from "../utils";
+import {
+    checkFilterString,
+    createDummyDataset,
+    createPhysicalDocument,
+    generateTempFileName,
+    validateDatasetName,
+    validateMemberName,
+} from "../utils";
 
 describe("Validate the member name", () => {
     it("Returns a warning about an empty name", () => {
-      const expectedReturnedValue = "MemberName must not be empty.";
-      const input = "";
-      const returnedValue = validateMemberName(input);
-      expect(expectedReturnedValue).toEqual(returnedValue);
+        const expectedReturnedValue = "MemberName must not be empty.";
+        const input = "";
+        const returnedValue = validateMemberName(input);
+        expect(expectedReturnedValue).toEqual(returnedValue);
     });
     it("Returns a warning about a big name", () => {
-      const input = "abcdefgji";
-      const expectedReturnedValue = "MemberName must not be more than 8 characters.";
-      const returnedValue = validateMemberName(input);
-      expect(expectedReturnedValue).toEqual(returnedValue);
+        const input = "abcdefgji";
+        const expectedReturnedValue = "MemberName must not be more than 8 characters.";
+        const returnedValue = validateMemberName(input);
+        expect(expectedReturnedValue).toEqual(returnedValue);
     });
     it("Returns a warning about a wrong first character in the name", () => {
-      const input = "3abfd";
-      const expectedReturnedValue = "Invalid Name " + input;
-      const returnedValue = validateMemberName(input);
-      expect(expectedReturnedValue).toEqual(returnedValue);
+        const input = "3abfd";
+        const expectedReturnedValue = "Invalid Name " + input;
+        const returnedValue = validateMemberName(input);
+        expect(expectedReturnedValue).toEqual(returnedValue);
     });
     it("Returns a warning about a wrong segment", () => {
-      const input2 = "tab(fd)";
-      const expectedReturnedValue = "Invalid Name " + input2;
-      const returnedValue = validateMemberName(input2);
-      expect(expectedReturnedValue).toEqual(returnedValue);
+        const input2 = "tab(fd)";
+        const expectedReturnedValue = "Invalid Name " + input2;
+        const returnedValue = validateMemberName(input2);
+        expect(expectedReturnedValue).toEqual(returnedValue);
     });
     it("Returns undefined", () => {
-      const input = "t3@y$C#";
-      const expectedReturnedValue = undefined;
-      const returnedValue = validateMemberName(input);
-      expect(expectedReturnedValue).toEqual(returnedValue);
+        const input = "t3@y$C#";
+        const expectedReturnedValue = undefined;
+        const returnedValue = validateMemberName(input);
+        expect(expectedReturnedValue).toEqual(returnedValue);
     });
-  },
-);
+});
 describe("Validate the dataset name", () => {
```

Comment:
The diff here is only related to the format?
As part of the cleanup I think we should uniform the formatting to avoid problem like that.

Reply:
I think it should be done in a different story about code styles.

Reply:
ok